.github/workflows/license-generation.yml

```diff
@@ -84,13 +88,13 @@ jobs:
   create-licenses:
 
     runs-on: ubuntu-latest
-  
+
     container:
-      image: zowe-docker-release.jfrog.io/ompzowe/zowecicd-license-base:latest
+      image: zowe-docker-snapshot.jfrog.io/ompzowe/zowecicd-license-base:test-timothy
```

Comment:
Do we still plan on using the `test-timothy` tag for the container image? No request for changes here, just wondering 😋

Reply:
Please revert the image to release/latest before merging in 👍